integration/spark-common/src/main/java/org/apache/carbondata/streaming/CarbonStreamRecordReader.java

```diff
@@ -115,7 +109,7 @@
 
   // vectorized reader
   private StructType outputSchema;
-  private ColumnarBatch columnarBatch;
+  private CarbonSparkVectorReader vectorProxy;
```

Comment:
CarbonSparkVectorReader interface not required. create a abstract class keeping common code.

Reply:
I will remove this interface, as we are moving CarbonstreamRecordReader to spark2 this interface will not be required

Reply:
Mainly because we cannot have any common api's as columnvector and columnarbatch package itself is changed, so we will not be able to extract it.